authorization-service/.env

```diff
@@ -0,0 +1 @@
+shurygindv1=somePassword
```

Comment:
password should be `TEST_PASSWORD` according to the task

Reply:
Thank you